tests/test_locking.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from unittest import mock
+import os
+import meg_runtime.locking as locking
+from meg_runtime.locking.lockFile import LockFile
+
+TOTAL_LOCKS_INITALIZED = 3
```

Comment:
Wouldn't it be easier to have a variable and increment/decrement how many locks you should have? So you don't have to count and change this number if the tests change?